manyfews/calculations/zentra.py

```diff
@@ -0,0 +1,238 @@
+    precip = []
+    wDirection = []
+    wSpeed = []
+    gustSpeed = []
+    vaporPressure = []
+    atmosPressure = []
+    airTem = []
+    maxPrecipRate = []
+    rhTemp = []
+    VPD = []
+    covertDate = []
+    RH = []
+
+    # Extract time stamp, Precipitation, solar, temperature, and humidity
+    for i in range(
+        len(zentraData["device"]["timeseries"][0]["configuration"]["values"])
+    ):
+
+        tStamp.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][0]
+        )  # time stamp data
+
+        solar.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][0][
+                "value"
+            ]
+        )  # solar radiation, 'unit':' W/m²'
+
+        precip.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][1][
+                "value"
+            ]
+        )  # Precipitation, 'unit':' mm'
+
+        airTem.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][7][
+                "value"
+            ]
+        )  # air temperature, 'unit'=' °C'
+
+        wDirection.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][4][
+                "value"
+            ]
+        )  # Wind Direction, 'units': ' °'
+
+        wSpeed.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][5][
+                "value"
+            ]
+        )  # wind speed, 'units': ' m/s'
+
+        vaporPressure.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][8][
+                "value"
+            ]
+        )  # vapor pressure, 'units': ' kPa'
+
+        atmosPressure.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][9][
+                "value"
+            ]
+        )  # Atmospheric Pressure, 'units': 'kPa'
+
+        VPD.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][14][
+                "value"
+            ]
+        )  # VPD, 'units': ' kPa'
+
+        rhTemp.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][13][
+                "value"
+            ]
+        )  # RH Senor Temp, 'units': ' °C'
+
+        gustSpeed.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][6][
+                "value"
+            ]
+        )  # Gust Speed, 'units': ' m/s'
+
+        maxPrecipRate.append(
+            zentraData["device"]["timeseries"][0]["configuration"]["values"][i][3][12][
+                "value"
+            ]
+        )  # Max Precip Rate, 'units': ' mm/h'
+
+        # convert time stamp to Date
+        ts = zentraData["device"]["timeseries"][0]["configuration"]["values"][i][
+            0
+        ]  # time stamp
+        tz = datetime.timezone(datetime.timedelta(hours=6))  # time zone
```

Comment:
Why do we have a hard-coded timezone? I think it might be better to use UTC time in all cases.